server/methods/core/groups.js

```diff
@@ -202,6 +214,6 @@ function setUserPermissions(users, oldGroup, newGroupData, shopId) {
   return affectedUsers.forEach(user => Roles.setUserRoles(user._id, newGroupData.permissions, shopId));
 }
 
-function getGroupPermissions(groups, name) {
-  return _.get(_.find(groups, { name }), "permissions", []);
+function getGroupPermissions(groups, groupId) {
+  return _.get(_.find(groups, { groupId }), "permissions", []);
```

Comment:
I think you can use Object prototype find here instead of lodash
Something like this perhaps
```
const group = groups.find(group => group.groupId === groupId);
return group.permissions || [];
```